charts/opentelemetry-operator/templates/certmanager.yaml

```diff
@@ -19,13 +19,17 @@ spec:
     {{- if .Values.admissionWebhooks.certManager.issuerRef }}
     {{- toYaml .Values.admissionWebhooks.certManager.issuerRef | nindent 4 }}
     {{- else }}
+    group: cert-manager.io
```

Comment:
@izazahamed-babji can you clarify that this is not a breaking change?

Reply:
@TylerHelmuth I don't think it is a breaking change as it is to make sure that the self signed issuer is picked and more of a patch.